lib/providers/services/eos_service.dart

```diff
@@ -366,4 +368,33 @@ class EosService {
 
     return request.encode();
   }
+
+  Future<Map<String, String>> fillActionPlaceholders(
```

Comment:
Where was all this code before - was this in our dart esr library and was pulled out from there? Or didn't exist before?

Reply:
Yes, our dart_esr package returns action with already filled placeholders; but new package properly returns raw action, and it's responsibility of application to parse and fill placeholdrers